en/docs/concepts/compliance/gdpr.md

```diff
@@ -0,0 +1,229 @@
+# General Data Protection Regulation
+
+## About GDPR compliance
+
+The General Data Protection Regulation (GDPR) is a legal framework formalized by the European Union (EU) in 2016. This regulation came into effect from 25, May 2018, and affects any organization that processes Personally Identifiable Information (PII) of individuals who live in Europe. Organizations that fail to demonstrate GDPR compliance are
```

Comment:
Shall we define PII or give some examples of it here? A basic user may not understand